superset/assets/javascripts/explore/components/controls/AnnotationLayer.jsx

```diff
@@ -0,0 +1,558 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { CirclePicker } from 'react-color';
```

Comment:
I think that `CompactPicker` is the best option of [those available](https://github.com/casesandberg/react-color) in `react-color`